charts/opentelemetry-collector/values.yaml

```diff
@@ -5,6 +5,10 @@
 nameOverride: ""
 fullnameOverride: ""
 
+# Define an OpenTelemetryCollector custom resource
+defineCustomResource: true
```

Comment:
This should be false by default.

Reply:
Resolved:
https://github.com/nhuray/opentelemetry-helm-charts/commit/28a2ad2d4c46f9808a2102b80a45f84cd92f3c3d